policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl

```diff
@@ -234,7 +234,6 @@
 
       - name: save deb
         uses: actions/upload-artifact@v4
-        if: {{`${{ matrix.golang_cross == '` }}{{.Branchvals.Buildenv}}{{`' }}`}}
```

Comment:
Thinking about this, removing this is not the right thing to do. When the config file is fixed and the templates applied, it will work correctly.